src/Resources/config/builder.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+use FOS\CKEditorBundle\Builder\JsonBuilder;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use function Symfony\Component\DependencyInjection\Loader\Configurator\service;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $services = $containerConfigurator->services();
+
+    $services->defaults()
+            ->public()
+            ->autowire();
+
+    $services->set('fos_ck_editor.builder.json_builder', JsonBuilder::class)
+            ->args([
+                service('property_accessor'),
+            ])
+    ;
```

Comment:
The indentation should use 4 spaces.
```suggestion
    $services->defaults()
        ->public()
        ->autowire();

    $services->set('fos_ck_editor.builder.json_builder', JsonBuilder::class)
        ->args([
            service('property_accessor'),
        ])
    ;
```